rtengine/iimage.h

```diff
@@ -98,6 +98,10 @@ class ImageDatas : virtual public ImageDimensions
         return "unknown";
     }
 
+    virtual bool isBW () const
```

Comment:
Inlining virtuals is always not so smart (except for the d'tor) and is not my style, either. I tried to adapt to the surrounding code.

But why would vague linkage bother something other than a library? Time to link?